docs/Server.md

```diff
@@ -418,6 +418,33 @@ Set a default
 Note that this is needed to offer the graceful "close" experience when
 using http2. Node core defaults this to `0`.
 
+<a name="framework-errors"></a>
+### `frameworkErrors`
+
++ Default: `null`
+
+Fastify provides default error handlers for the most common use cases.
+Using this option it is possible to override one or more of those handlers with custom code.
+
+*Note: Only `FST_ERR_BAD_URL` is implemented at the moment.*
+
+```js
+const fastify = require('fastify')({
+  frameworkErrors: function (error, req, res) {
+    if (error instanceof FST_ERR_BAD_URL) {
+      const headers = {
+        'Content-Type': 'text/plain'
+      }
+      res.writeHead(400, headers)
```

Comment:
```suggestion
      res.code(400)
```

having a reply object is easier to write code 😀 (the content-type is added by default)

Reply:
> @alemagio , please don't hate me  : last suggestion to fix the readme

Don't worry, I am happy to have such feedback. I'm here to learn and I am doing it for free.

I'll fix it asap